src/test/resources/db/performance/insert_bulk_assignments.sql

```diff
@@ -0,0 +1,96 @@
+-- Postgres Function to insert bulk assignment records
+drop type if exists roleCategories;
+create type roleCategories as enum('PROFESSIONAL', 'JUDICIAL','LEGAL_OPERATIONS');
+
+drop type if exists professionalRoles;
+create type professionalRoles as enum('petitioner''s-solicitor', 'respondent''s-solicitor','ls-solicitor', 'legal-representative', 'barrister', 'cafcass-solicitor', 'external-professional-solicitor', 'la-barrister',
+                          'managing-local-authority', 'solicitor', 'solicitor-a', 'solicitor-b',
+                          'solicitor-c', 'solicitor-d', 'solicitor-e', 'solicitor-f', 'solicitor-g', 'solicitor-h',
+                          'solicitor-i', 'solicitor-j');
+
```

Comment:
Role names should be updated like this
[LAMANAGING][LASOLICITOR][SOLICITOR][SOLICITORA][SOLICITORB]

Reply:
This was discussed with Aashish earlier and later we used names instead of references.

Reply:
Yes we  need to update the role name as per new ccd-case role PR.

Reply:
Refer this File: https://github.com/hmcts/am-role-assignment-service/blob/29f33e1e8400c60f3c8ca9a069d16900888c7f8a/src/main/resources/role.json

Reply:
done